src/services/mongo/pagination.ts

```diff
@@ -0,0 +1,33 @@
+export default class Pagination {
+    private sortBy: string;
+    private order: "ASC" | "DESC";
+    private page: number;   // Starts from 1
+    private pageSize: number;
+
+    // Defaults
+    private constructor(sortBy:string = "_id", order:"ASC" | "DESC" = "ASC", page:number = 1, pageSize:number = 100) {
```

Comment:
```suggestion
    private constructor(
        sortBy: string = "_id",
        order: "ASC" | "DESC" = "ASC",
        page: number = 1,
        pageSize: number = 100
    ) {
```

Primarily because this will undoubtedly expand